### ISSUE TITLE
ERROR #85901  for "allShopifyProduct"

### ISSUE THREAD

**Issue by foamingmonkey** [AlexanderProd/gatsby-shopify-starter#60]
Hey guys look like I keep getting these error   and it looks like it is in fact connecting to the site but still giving me these errors 
1: **error** an error occurred while sourcing data
There was an error in your GraphQL query:

2. Cannot query field "allShopifyProduct" on type "Query". Did you mean "allShopifyBlog", "allShopifyArticle", "allShopifyShopPolicy", or "allShopifyCollection"?

The site has 400+ products, 20 blogs, etc so not sure why I'm getting the Error ? is it that Shopify changed the tag or is it something else

**Comment by AlexanderProd** [comment]
Do you have used the right API credentials? Looks like gatsby can’t access your products.

Am 02.10.2020 um 22:30 schrieb anthony <notifications@github.com>:

﻿

Hey guys look like I keep getting these error and it looks like it is in fact connecting to the site but still giving me these errors
1: error an error occurred while sourcing data
There was an error in your GraphQL query:

  1.  Cannot query field "allShopifyProduct" on type "Query". Did you mean "allShopifyBlog", "allShopifyArticle", "allShopifyShopPolicy", or "allShopifyCollection"?

The site has 400+ products, 20 blogs, etc so not sure why I'm getting the Error ? is it that Shopify changed the tag or is it something else

**Comment by AlexanderProd** [comment]
Did you use the right API credentials for the Storefront API? Looks like Gatsby can’t access the products.

**Comment by foamingmonkey** [comment]
yes, 100% the one all the way on the bottom of private apps right?  i even changed the   SHOP_NAME=graphql on both and nada just errors .. like it will run but error out on templates/ProductPage/index.js

**Comment by foamingmonkey** [comment]
found the error apparently this only works with a real domain, not a dev domain or dev site.. the site has to be live via Bluehost,GoDaddy etc to run correctly.. anyone has a workaround?

**Comment by AlexanderProd** [comment]
@foamingmonkey What do you mean it doesn’t work with a dev domain or dev site.

**Comment by foamingmonkey** [comment]
Hey Alex, you know on Shopify you can make your own Development site before you publish well that didn't work for me, I called it something like **myspeedbuild2/myshopify.com** and that errored out regardless of what I tried  ( again I had products, blog, etc all set up on the dummy site) . So I grabbed my client's site a live site and tested to see if it was just the Domain meaning the site has to be live for the gatsby to pull from and connected.   Well, that is my best guest but IDK I'm new to gatsby, to be honest.

With my client's site, it connected however since my client's site has over 3 million photos it froze out the terminal

**Comment by ocmartineau** [comment]
Hi there!

I am also running into this error though I haven't added in my own API Keys and credentials. Getting it from just installing and running out of the box from this repo. Thoughts?

**Comment by AlexanderProd** [comment]
@ocmartineau I've just tried it by cloning the repo fresh and running `yarn start` as well as `yarn build` but it worked for me.
What environment are you in? Node version, OS, anything out of the ordinary?

**Comment by arjayosma** [comment]
I am also experiencing the same issue with a test store.
Our test store has a lot of collections and products.
I saw a related issue here: https://github.com/angeloashmore/gatsby-source-shopify/issues/24

The error is most likely caused during the initialization/build of the application. Gatsby is downloading/fetching the data from the Shopify Storefront API. It'll throw a timeout (despite providing a lower `paginationSize`).
I strongly believe this should be handled by the `gatsby-source-shopify` plugin.

**Comment by krichey15** [comment]
My site has been live for a few days now. I added web hooks so it would build on Product: Addition, Deletion, and Update. But today it through this error when building with Netlify

error There was an error in your GraphQL query:
3:07:51 PM: Cannot query field "allShopifyProduct" on type "Query".
3:07:51 PM: If you don't expect "allShopifyProduct" to exist on the type "Query" it is most likely a typo.

**Comment by kaueburiti** [comment]
I'm getting the same kind of problem but with `allReview`
